_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,123 @@
+
+We found that keeping on track of all pods is starting to become an issue, for example yesterday 18 new pods were added to trunk. We wanted to find an alternative way to keep on top of new pods, so [we created](/Creme-de-la-Pods) [@CremeDeLaPods](https://twitter.com/CremeDeLaPods) as a way of keeping track of the highest quality _new_ pods.
+
+We worked with Rails Girls Summer of Code this year ...
+
+## Web Properties
+
+A problem close to my heart is discoverability. We want to lower the barriers to entry for everyone to participate in OSS, and making people's projects easily discovered is an important part of this.
+
+The CocoaPods web presence has had a lot of work devoted to it in the last year, the most obvious is that we now have individual pages for a pod. These pod pages pull in nearly every interesting piece of metadata around the pod, making it significantly easier to browse and compare pods.
+
+We introduced a project we've had been working on for over a year, [CocoaPods Quality Indexes](https://guides.cocoapods.org/making/quality-indexes.html) (QIs) an evolving attempt at defining quality in software. These QIs are used throughout the web properties to provide feedback and to sort pods.
+
+The QIs provide a positive feedback loop, showing you ways in which you can game the system by improving your library. By focusing on showing you ways to improve rather than penalising discouraged behaviour I hope we can continue to foster a welcome community.
+
+To expand on that further and allow people to have a better sense of their the state of their libraries, we created Pod Author Pages. Here's a great example from [Andrea Mazzini](https://cocoapods.org/owners/734) making it really easy to understand his impact on the community.
+
+[pic]
+
+Another multi-year project that we shipped this year was [CocoaPods Stats](/Stats/). Stats offers a way of knowing how many downloads a pod has, and far more interestingly the number of targets it has been integrated with. This is something I haven't found in other dependency managers, as it's a really hard problem to solve. We couldn't have shipped this alone, so a big thanks to [Segment](https://segment.com) for sponsoring a lot of the infrastructure here.
+
+While the number of pods available via CocoaPods has increased in the last year, the amount of servers we need for search has not. [Florian Hanke](https://github.com/floere) who has been working with us for years on search continues to refuse to throw more money at our search server. His work has made it possible for us to deal with doubling our search traffic, support more metadata, add different sort types and allow scoping to all the new platforms Apple keep shipping.
```

Comment:
Just an FYI: it hasn’t just been ‘for years’, he has been there from almost the very start. At that time it was just him and me :)